front-end/src/components/Students/Table/index.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState, useContext, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+// FontAwesome library
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisVertical } from '@fortawesome/free-solid-svg-icons';
+
+// Context
+import { CurrentUserContext } from 'context/AppProvider';
+
+// Service
+import { getTeachersSubscription, deleteSubscription } from 'services/subscriptionService';
+
+// Components
+import NoStudents from '../NoStudents';
+import ChangeLevel from 'components/ChangeLevel/ChangeLevel';
+
+// Images
+import teacher from 'assets/images/teacher1.jpg';
+
+// Styles
+import styles from './Table.module.scss';
+import { Menu, MenuItem } from '@mui/material';
+
+const Table = () => {
```

Comment:
Why did you create a new table? As I know we already have a table for subscriptions.